Application/Client/src/app/_common/services/headers.service.ts

```diff
@@ -0,0 +1,24 @@
+import {Injectable} from '@angular/core';
+import {Headers} from '@angular/http';
+import {CookieService} from 'angular2-cookie/core';
```

Comment:
pls check is this proper import as there is not entry of angular2-cookie in package.json